src/libsyntax/ext/tt/quoted.rs

```diff
@@ -50,11 +50,17 @@ pub struct SequenceRepetition {
     /// The optional separator
     pub separator: Option<Token>,
     /// Whether the sequence can be repeated zero (*), or one or more times (+)
-    pub op: KleeneOp,
+    pub op: KleeneToken,
```

Comment:
```suggestion
    pub kleene: KleeneToken,
```

Reply:
Thanks, I wasn't satisfied with the current naming duplication.